source/common/http/conn_manager_impl.h

```diff
@@ -90,6 +90,8 @@ class ConnectionManagerImpl : Logger::Loggable<Logger::Id::http>,
 
   TimeSource& timeSource() { return time_source_; }
 
+  std::shared_ptr<StreamInfo::FilterState> filter_state() const { return filter_state_; }
```

Comment:
nit: `filterState()` (probably will need modification to deal with lazy init, not sure)